assessment.md

```diff
@@ -12,82 +12,447 @@
+  * I = Rarely respectful and inclusive of teammates
+  * A = Sometimes respectful and inclusive of teammates
+  * G = Regularly respectful and inclusive of teammates
+  * E = Always respectful and inclusive of teammates
 * Decisions
+  * N = Never participate in the decision making process and share your opinion.
+  * I = Rarely participate in the decision making process and share your opinion
+  * A = Sometimes participate in the decision making process and share your opinion
+  * G = Regularly participate in the decision making process and share your opinion
+  * E = Always participate in the decision making process and share your opinion.
 * Postmortems
+  * N = Never participate the analysis or discussion after the work is done
+  * I = Rarely participate the analysis or discussion after the work is done
+  * A = Sometimes participate the analysis or discussion after the work is done
+  * G = Regularly participate the analysis or discussion after the work is done
+  * E = Always participate the analysis or discussion after the work is done
 * Conflicts
-* Infractions
+  * N = Never handles differences with teammates in mature manner
+  * I = Rarely handles differences with teammates in mature manner
+  * A = Sometimes handles differences with teammates in mature manner
+  * G = Regularly handles differences with teammates in mature manner
+  * E = Always handles differences with teammates in mature manner
 
 ### Interactions
 
 * Interaction with the customer
+  * N = Does not respect the customer and rejects all their ideas
+  * I = Respect customer but rejects most of their ideas
+  * A = Respect customer but fails to build a system around their needs
+  * G = Respect customer and builds a system around their needs but fails
+     to clearly present your system to your customer
+  * E = Respect customer and builds a system around their needs while being
+     able to clearly present your system to them
 * Interaction with the team leaders
+  * N = Impolite and impatient
+  * I = Polite, impatient, and does not respond to team leaders
+  * A = Polite, patient, responsive to team leaders but does not have clear
+  requests
+  * G = Polite, patient, responsive to team leaders and clear requests
+  * E = Polite, patient, responsive to team leaders, clear requests,
+  and provides progress updates
 * Interaction with the course instructor
-* Interaction with the teaching assistants
+  * N = Communicated with the instructor badly, did not follow
+     the project and course requirements
+  * I = Communicated with the instructor poorly, followed part
+     of the project and course requirements
+  * A = Communicated with the instructor well, followed most
+     of the project and course requirements
+  * G = Communicated with the instructor well, followed all
+     project and course requirements
+  * E = Communicated with the instructor excellently, followed
+     all project and course requirements
+* Interaction with the technical leads
```

Comment:
There are a couple of issues with tense in this section.

I would suggest the following:

N: ...used hate speed, ignored technical leads...
G: ...and tried to solve...
E: ...politely shared own opinions and worked with them...

Reply:
fixed by https://github.com/huangs1/assessment-professional-skills/pull/43

Reply:
@sutterj Thanks so much for pointing it out!